libs/s25main/world/GameWorldView.cpp

```diff
@@ -484,7 +490,7 @@ void GameWorldView::DrawConstructionAid(const MapPoint& pt, const DrawPoint& cur
         // Draw building quality icon
         bm->DrawFull(curPos);
         // Show ability to construct military buildings
-        if(GetWorld().GetGGS().isEnabled(AddonId::MILITARY_AID))
+        if(GetWorld().GetGGS().getSelection(AddonId::MILITARY_AID) > 0)
```

Comment:
Why this change? `isEnabled` already does exactly this.

Reply:
Because I've changed setting to list instead of bool, so that there are now 3 states of it: "off", "only construction aid", and "construction and attack aid". I assumed that `isEnabled` is only part of bool-based plugins, perhaps I was wrong about it. I'll check that out tomorrow

Reply:
`isEnabled` is a shortcut for `isNotDefault`, i.e. the default for an addon should be the original S2 behavior, i.e. "disabled"
Hence the name.

Reply:
fixed!